README.md

```diff
@@ -18,6 +18,10 @@ RCOS offers a huge community of open source developers skilled in nearly any pla
 
 RCOS is offered for credit at RPI and fulfills a 4000 level technical elective.
 
+# How to join RCOS
+
+If your an RPI student, hop in on our first meeting of the semester or join [our slack](https://rcos.slack.com) and message a faculty coordinator (@mskmoorthy or @turnew2). If you're looking to get involved with RCOS as a member outside of RPI, email a [coordinator](mailto:richiejoeyoung@gmail.com).
```

Comment:
> If your an RPI student

you're

> email a [coordinator](mailto:richiejoeyoung@gmail.com).

Do we want to hardcode Richie's email as the default contact? I don't particularly like putting his email in a public repository... Though maybe it's public already, so putting it one more place doesn't matter so much. It might be cool if we had like a coordinators email which would forward to the coordinators at any given time. E.g. coordinators@rcos.io would forward email to Kiana, Richie and I.